snap/snapcraft.yaml

```diff
@@ -0,0 +1,47 @@
+name: newsboat
+version: git
+summary: An RSS/Atom feed reader for text terminals
+description: |
+  Newsboat is a fork of Newsbeuter, an RSS/Atom feed reader for the text 
+  console. The only difference is that Newsboat is actively maintained 
+  while Newsbeuter isn't.
+grade: stable
+confinement: strict
+
+apps:
+  newsboat:
+    command: usr/local/bin/newsboat
+    environment:
+      LOCPATH: "$SNAP/usr/lib/locale"
+    plugs:
+      - network
+      - home
+  podboat:
+    command: usr/local/bin/podboat
+    aliases: [podboat]
+    environment:
+      LOCPATH: "$SNAP/usr/lib/locale"
+    plugs:
+      - network
+      - home
+
+parts:
+  newsboat:
+    plugin: make
+    source: .
+    build-packages: 
+      - pkg-config
+      - libsqlite3-dev
+      - libcurl4-openssl-dev
+      - libxml2-dev
+      - libstfl-dev
+      - libjson-c-dev
+      - libncursesw5-dev
+      - asciidoc
+      - libssl-dev
+      - gettext
+    stage-packages:
+      - libxml2
+      - librtmp1
```

Comment:
I don't think Newsboat uses that library, even transitively. I assume you added it after you've run into some error?

Reply:
I don't recall exactly what triggered me adding this. Sorry. It likely would have been an error triggered by a feed not loading. I'm not sure which feed was at fault here. I'm happy to rebuild without it and see what breaks, if you'd prefer.

Reply:
No worries then. I'll revisit this myself when Snap requires me to list all stage packages.